aksel.nav.no/website/pages/eksempler/timeline-intern/dynamic.tsx

```diff
@@ -0,0 +1,278 @@
+            }}
+          />
+          <Button
+            icon={<ChevronRightIcon title="Neste periode" />}
+            variant="secondary-neutral"
+            size="small"
+            onClick={() => {
+              /**
+               * We move the window with 50% the total width to make it easier
+               * to navigate through the timeline.
+               * This is a design choice, and can be adjusted to fit your needs.
+               */
+              updateWindow("next", parseInt(windowSize) * 0.5, windowSize);
+            }}
+          />
+        </HStack>
+        <ToggleGroup
+          variant="neutral"
+          size="small"
+          value={windowSize}
+          onChange={(value) => {
+            setWindowSize(value as WindowSize);
+            updateWindow("current", parseInt(value), value as WindowSize);
+          }}
+        >
+          <ToggleGroup.Item value="2" label="2 mnd" />
+          <ToggleGroup.Item value="4" label="4 mnd" />
+          <ToggleGroup.Item value="8" label="8 mnd" />
+        </ToggleGroup>
+      </HStack>
+
+      <Timeline
+        startDate={currentWindow.start}
+        endDate={currentWindow.end}
+        id="timeline-dynamic"
+        aria-controls="timeline-toolbar"
+      >
+        <Timeline.Row
+          label="Person A"
+          icon={<PersonSuitFillIcon aria-hidden fontSize="1.5rem" />}
+        >
+          {rows.rowOne.map((p) => (
+            <Timeline.Period
+              key={p.start.toString()}
+              start={p.start}
+              end={p.end}
+              status={p.status}
+              icon={p.icon}
+              statusLabel={p.statusLabel}
+            />
+          ))}
+        </Timeline.Row>
+        <Timeline.Row
+          label="Person B"
+          icon={<PersonSuitFillIcon aria-hidden fontSize="1.5rem" />}
+        >
+          {rows.rowTwo.map((p) => (
+            <Timeline.Period
+              key={p.start.toString()}
+              start={p.start}
+              end={p.end}
+              status={p.status}
+              icon={p.icon}
+              statusLabel={p.statusLabel}
+            />
+          ))}
+        </Timeline.Row>
+        <Timeline.Row
+          label="Person C"
+          icon={<PersonSuitFillIcon aria-hidden fontSize="1.5rem" />}
+        >
+          {rows.rowThree.map((p) => (
+            <Timeline.Period
+              key={p.start.toString()}
+              start={p.start}
+              end={p.end}
+              status={p.status}
+              icon={p.icon}
+              statusLabel={p.statusLabel}
+            />
+          ))}
+        </Timeline.Row>
+      </Timeline>
+    </VStack>
+  );
+};
+
+/**
+ * @internal Dont copy this code to your own project.
+ * This code is only for generating example data
+ *
+ * Code for generating time periods in the example.
+ * This part would normally not be part of your application code,
+ * and fetch data from an API or similar.
```

Comment:
There is some overlap between the two paragraphs, so I think it can be simplified.
```suggestion
 * @internal Don't copy this code to your own project.
 * This is only for generating time periods for the example.
 * You would normally fetch this data from an API or similar.
```